mpact_api/telegram_bot/settings.py

```diff
@@ -20,64 +20,69 @@
 # See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('SECRET_KEY')
+SECRET_KEY = os.environ.get("SECRET_KEY")
```

Comment:
Just curious: Are you using an automated formatter, like Black?

Reply:
Yes, I'm using Black formatter.